packages/smart/contracts/chainlink/TheRundownChainlink.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+import "@chainlink/contracts/src/v0.6/ChainlinkClient.sol";
+
+contract TheRundownChainlink is ChainlinkClient {
+    uint256 public score;
+    address private oracle;
+    bytes32 private jobId;
+    uint256 private fee;
+
+    constructor() public {
+        setPublicChainlinkToken();
+        oracle = 0x56dd6586DB0D08c6Ce7B2f2805af28616E082455;
+        jobId = "dbb65efc02d34cddb920eca1bec22ade";
+        fee = 0.1 * 10**18; // 0.1 LINK
```

Comment:
I'm wary of decimals in solidity. I imagine this works since the numbers are fairly small but I'd do `10**17` instead.